copy-music.sh

```diff
@@ -40,37 +40,36 @@ fi
 # functions {{{1
 # fuctions for user interaction and general script execution {{{2
 usage () { # {{{3
-  echo $PROG src target
-  echo Convert all music files in src to ogg files into target.
+  echo "$PROG src target"
+  echo "Convert all music files in src to ogg files into target."
 }
 
 die () { # {{{3
-  ret=$1
   shift
-  echo $PROG: $@ >&2
-  exit $ret
+  echo "$PROG: $*" >&2
+  exit $1
 }
 
 handle_signal () { # {{{3
   if $CONTINUE; then
     TIMESTAMP=$(date +%s)
     CONTINUE=false
-    echo Let me just finish converting $file ... >&2
+    echo "Let me just finish converting $file ..." >&2
   elif [[ $(($TIMESTAMP + 10)) -lt $(date +%s) ]]; then
     echo 'OK, if you force me.  Engines STOP!' >&2
     exit 10
   else
-    echo Let me just finish converting $file ... >&2
+    echo "Let me just finish converting $file ..." >&2
   fi
 }
 
 # filesystem interaction {{{2
 resolve_symlinks () { # {{{3
-  local file=$1
-  while [[ -l $file ]]; do
-    file=$(readlink $file)
+  local file="$1"
```

Comment:
Ooh, I knew this with variable assignments (I must have gotten carried away, since you had a _lot_ of unprotected strings back then lol) but had no idea `case` did it too! That's a useful tip. Thanks.